### PR TITLE
Complete fixes

### PR DESCRIPTION
**Pull request by denscholar**
Added the following files which include the following functionalities:

- Linter files
- Javascript files
- CSS and HTML files
- Speakers and mentors and served dynamically from the javascript file

### CODE REVIEW

File: .github/workflows/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: Linters
```
Comment (julie-ify) [inline .github/workflows/workflows/linters.yml:1]: - [ ] Hi there! There is a name error in this directory which makes your linters check not show up on Github. Kindly make the corrections as the name for this directory is `.github/workflows/linters.yml` and not `.github/workflows/workflows/linters.yml` 😄.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @denscholar, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

## You have submitted a project, that does not have a Github action showing the linter checks. I left a comment for you below.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by HENRYKC24 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUESTED:recycle:
Changes requested
## Hi @denscholar :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] There are some steps yet to be taken for your project to be complete. I think you forgot them.
1. You need to record a video describing your project as it is in the requirement:point_down:
image
2. The designer of the template needs to be acknowledged in the readme file.
image
Please, let's mention the name in the acknowledgment.
3. You also forgot to deploy your app with GitHub pages and also add the link to the readme file.
**I strongly advise that you use the[ Microverse readme template](https://github.com/microverseinc/readme-template) in addition to yours**.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+        <div class="program-section-items">
+          <img src="./images/icons/forum.png" alt="bg-1" />
+          <h3>Join Forum</h3>
+          <p>
+            You can leverage on our mentors and mentees forum for a quick
+            response.
+          </p>
+        </div>
+        <div class="program-section-items">
+          <img src="./images/icons/ignite.png" alt="bg-1" />
```
[the remark this answers is not in the document]
Comment (denscholar) [reply index.html:118]: @HENRYKC24 Noted with thanks.

File: app.js [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+const seeMoreText = document.createElement('h3');
+seeMoreText.textContent = 'More';
+button.appendChild(seeMoreText);
+
+const icon = document.createElement('i');
+icon.className = 'fas fa-chevron-down';
+seeMoreText.appendChild(icon);
+
+function mentorsData() {
```
[the remark this answers is not in the document]
Comment (denscholar) [reply app.js:103]: @HENRYKC24 Noted with thanks

File: app.js [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+  for (let i = 0; i < data.length; i++) {
```
[the remark this answers is not in the document]
Comment (denscholar) [reply app.js:104]: @HENRYKC24 Noted with thanks.

**Comment by denscholar** [comment]
Presentation link:
https://www.loom.com/share/68530c6898c14d3a84789c9761bf976a

**Review by lily-coder — APPROVED** [review]
Hi @denscholar , 

Thank you for implementing the changes as requested by the previous reviewers. In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 

image
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**